src/components/Dashboard/StaffNotation.vue

```diff
@@ -88,7 +88,7 @@
                 <span class="app-blue">Record Conversion</span>
               </v-list-item-title>
             </v-list-item>
-            <template v-if="isFirm">
+            <template v-if="isFirm || isCoop || isBComp">
```

Comment:
"remove condition for put back on filing for BEN and CP"

Isn't this adding the conditions?

Reply:
sorry poor naming